packages/webapp/package.json

```diff
@@ -16,7 +16,7 @@
     "build-storybook": "storybook build",
     "test-storybook": "test-storybook",
     "precommit": "lint-staged",
-    "test": "vitest"
+    "test": "vitest --environment happy-dom"
```

Comment:
Had to add this here because I use `i18n` in the function and it doesn't work unless it runs in a browser environment.